Makefile

```diff
@@ -1,20 +1,13 @@
 SENTRY_PATH := `python -c 'import sentry; print sentry.__file__.rsplit("/", 3)[0]'`
 UNAME_S := $(shell uname -s)
 
-develop: setup-git install-yarn
-	pip install "pip>=9,<10"
+develop: ensure-pip-version setup-git install-yarn
 	pip install -e git+https://github.com/getsentry/sentry.git#egg=sentry[dev,optional]
-	pip install -e ".[tests]"
+	pip install --no-use-pep517 -e ".[tests]"
 	yarn install
 
-install-yarn:
-	@echo "--> Installing Node dependencies"
-	@hash yarn 2> /dev/null || npm install -g yarn
-	# Use NODE_ENV=development so that yarn installs both dependencies + devDependencies
-	NODE_ENV=development yarn install --ignore-optional
-
-install-tests: develop
-	pip install .[tests]
+ensure-pip-version:
+	pip install "pip>=19.2.1,<20"
```

Comment:
Can we lock to a specific pip version

Reply:
I feel like a major pin > 19.2 is good enough, but sure.

Reply:
It's probably fine, but we've been pinning to explicit versions everywhere else.

Reply:
getsentry is >=19,<20, and sentry doesn't have pip pinned (just at a cursory glance), so actually i'm just gonna leave this as is.